yarn.lock

```diff
@@ -17175,6 +17189,35 @@ webpack@4.42.1:
     watchpack "^1.6.0"
     webpack-sources "^1.4.1"
 
+webpack@4.43.0:
```

Comment:
There are now two copies of `webpack` in the repo, the older one due to https://github.com/zeit/next.js/blob/ad16b8c8233850b939e9de9b24a8ab7851976a52/packages/react-refresh-utils/package.json#L27 was it intentional or an oversight?